src/antares/model/study.py

```diff
@@ -194,7 +211,13 @@ def service(self) -> BaseStudyService:
         return self._study_service
 
     def read_areas(self) -> list[Area]:
-        return self._area_service.read_areas()
+        """
+        Syncronize the internal study object with the object written in an antares study
```

Comment:
typo: Synchronize, also you can either remove the Return: of finish your sentence :)